OOModel/src/declarations/Class.cpp

```diff
@@ -228,4 +239,22 @@ Class* Class::expressionToClass(Expression* expr)
 	return nullptr;
 }
 
+QSet<Class*> Class::directSubClasses()
+{
+	auto top = root();
+	//Find all the classes where this method is subclasses
+	QSet<Class*> result;
+	QList<Model::Node*> toCheck;
+	toCheck.append(top);
+	while (!toCheck.isEmpty())
```

Comment:
As the class could be referenced from anywhere within the program, I don't have any really good ideas. The only idea is to store whenever such a reference is created in the class, but that means also updating it on changes/deletions.